apps/condo/domains/acquiring/schema/RegisterMultiPaymentForInvoicesService.test.js

```diff
@@ -336,6 +343,72 @@ describe('RegisterMultiPaymentForInvoicesService', () => {
                 }),
             ]))
         })
+
+        test('register multiPayment for invoice from different organization', async () => {
+            const utils1 = new TestUtils([ResidentTestMixin])
+            const utils2 = new TestUtils([ResidentTestMixin])
+            await utils1.init()
```

Comment:
you can use better naming to help another developers to understand tests faster
for example,
const utils1 = new TestUtils([ResidentTestMixin]) can be 
const organizationUtils = new TestUtils([ResidentTestMixin])

and utils2  can be anotherOrganizationUtils (@SavelevMatthew request  )